pkg/handler/defi.go

```diff
@@ -93,7 +93,7 @@ func (h *Handler) InDiscordWalletWithdraw(c *gin.Context) {
 		return
 	}
 
-	c.JSON(http.StatusOK, gin.H{"data": res})
+	c.JSON(http.StatusOK, response.CreateResponse(res, nil, nil, nil))
 }
 
 // InDiscordWalletBalances     godoc
```

Comment:
do we need to wrap this response? (line 124)

Reply:
this is one of my afraid, too many ridiculous response that i dont know if it will affect bot side

Reply:
for now i think just do simple things first